uint8 STATUS_SUCCESS          = 10
uint8 STATUS_FAILURE          = 20  # General failure (in case not known).
# uint8 STATUS_NOT_WRITABLE     = 21  # Use this status once we can detect that saving a configuration failed because of this.
uint8 STATUS_INVALID_FILENAME = 22

string    config_file_path    # Path to the file to which the configuration is written.
bool      set_active          # Boolean indicating whether the newly saved configuration should be used.
---
uint8     status              # Return status indicating success / failure.
string    message             # Informational, e.g. for error messages.
